### ISSUE TITLE
[Accesibility Hidden view on Master Detail Page Bug] 

### ISSUE THREAD

**Issue by pamela032709** [xamarin/Xamarin.Forms#7676]
### Description
I think this is related with this #7676 
When voice over is activated and you are using a MasterDetailPage,  menu content still visible for the reader even when this master-detail page is not present or it is hidden.  Additional information for that respective page gets all mix-up and this is very confusing and problematic for voice over users. This happens on Android and IOS
### Steps to Reproduce

1. Download the following sample
 https://docs.microsoft.com/en-us/samples/xamarin/xamarin-forms-samples/navigation-masterdetailpage/
2. Activate voice over 
3. Navigate to any page from the menu and you will be able to hear any item on that list even when is not present.

### Expected Behavior
When the master-detail page is closed or hidden, voice-over should be able to read its content
### Actual Behavior
Is reading everything that is on the menu list even when is closed
### Basic Information

- Version with issue:
 All versions - This has never been addressed 
- Last known good version:
- IDE: Android - IOS
- Platform Target Frameworks: 
  - iOS:  
  - Android:  

- Android Support Library Version: 
- NuGet Packages:
     Xamarin Forms 4.2.0.778463
- Affected Devices:

### Reproduction Link
https://docs.microsoft.com/en-us/samples/xamarin/xamarin-forms-samples/navigation-masterdetailpage/

**Comment by samhouts** [comment]
Duplicate of #4220
